release/scripts/startup/bl_ui/properties_data_camera.py

```diff
@@ -151,8 +151,8 @@ def draw(self, context):
         col.prop(cam, "lod_factor", text="Distance Factor")
 
 
-class DATA_PT_frustum_culling(CameraButtonsPanel, Panel):
```

Comment:
I made the UI very quick, i certainly didn't choose the best presentation... @youle31 : Feel free to rework it.